etc/arno-iptables-firewall/plugins/parasitic-net.conf

```diff
@@ -23,36 +23,66 @@ ENABLED=0
 # (IPv4 Only)
 # ------------------------------------------------------------------------------
 
-# Specify which "clients" on your subnet that are allowed to use this device
-# as a gateway.  Define IPv4 addresses, IPv4 address ranges or a small CIDR.
-# Example: "192.168.1.10 192.168.1.15 192.168.1.20-30"
-# Must be defined, there is no default.
+# Specify which (external) network interfaces and/or interface-IPS should have
+# parasitic SNAT enabled. Either specify the interface or interface~IP (for eg.
+# interfaces with multiple IP addresses). Multiple interfaces should be space
+# separated
+# ------------------------------------------------------------------------------
+PARASITIC_NET_IF="$EXT_IF"
```

Comment:
Personally I think it is better to not make variable assignments via the configs, but rather document an undefined variables's default.  So ...
```
PARASITIC_NET_IF=""
```
and in the plugin
```
if [ -z "$PARASITIC_NET_IF" ]; then
  PARASITIC_NET_IF="$EXT_IF"
fi
```
or only use the first `EXT_IF` as the default, using a `for` loop and `break` after first assignment.

and remove the `PARASITIC_NET_IF` sanity check.